src/app/shared/image/image-upload.component.ts

```diff
@@ -33,6 +38,8 @@ export class ImageUploadComponent implements DoCheck {
   @Input() purpose: string;
   @Input() minWidth = 144;
   @Input() minHeight = 144;
+  @Input() maxWidth = 144;
+  @Input() maxHeight = 144;
```

Comment:
The series thumbnail image isn't working. Can we maybe not have a default max or something? Or maybe just in the validation below check for either suggestSize or min/max?
<img width="683" alt="screen shot 2017-09-07 at 5 52 56 pm" src="https://user-images.githubusercontent.com/2250413/30188702-939d5ba6-93f5-11e7-9da5-de6f0933d0eb.png">

Reply:
oooh, good catch!